comment/comment_repository.go

```diff
@@ -115,7 +115,7 @@ func (m *GormCommentRepository) Delete(ctx context.Context, commentID uuid.UUID,
 	}
 	// fetch the id and parent id of the comment to delete, to store them in the new revision.
 	c := Comment{}
-	tx := m.db.Select("id, parent_id").Where("id = ?", commentID).Find(&c)
+	tx := m.db.Select("id, parent_id, parent_comment_id").Where("id = ?", commentID).Find(&c)
```

Comment:
@michaelkleinhenz Why is this change required ?

Reply:
The revision entry on delete then contains the original parentCommentID. I modelled that the same way the parentID (for the WorkItem parent) is modelled: this reference is also included in the delete revision entry. The markdown and other payload is omitted, though. I think keeping the references in the revision entry makes sense.